### DISCUSSION TITLE
Get location

### DISCUSSION THREAD

**Discussion by Freshblood1121** [def-studio/telegraph#555] [Q&A]
Hi, I don't understand how to work with location.
I use the ->requestLocation() method, but when the user sends it, I can’t work with it🤷‍♂️

![123](https://github.com/defstudio/telegraph/assets/59627636/06c53aa7-5f39-4f06-a108-4172f59223ab)

![456](https://github.com/defstudio/telegraph/assets/59627636/f53f8d58-510e-41c7-8e64-a3e8c8b5f20a)

**Comment by Freshblood1121** [comment]
Where and how can I get it latitude and longitude?

**Comment by fabio-ivona** [comment]
In order to get the request location result, you should set up a webhook and handle the response
